shared_libraries/orca_shared/database/use_cases/create_postgres_connection_uri.py

```diff
@@ -79,4 +79,4 @@ def _create_connection_uri(logger: logging.Logger, **kwargs: Any) -> str:
         URI for connecting to the database.
```

Comment:
Change the return type.

Reply:
changed to 
`URL for connecting to the database. `

Reply:
changed to URL